### DISCUSSION TITLE
欢迎使用 HowToCook discussion

### DISCUSSION THREAD

**Discussion by Anduin2017** [Anduin2017/HowToCook#1083] [General]
在这里可以畅所欲言交流厨艺、分享你做的菜、讨论仓库的需求。

但对于发现的菜谱的问题，或对现有菜谱进行改进，建议使用 issue 和 pull request 功能。

### 在使用 discussion 功能时，用户应当遵守：

5.1.1《中华人民共和国保守国家秘密法》

5.1.2《中华人民共和国著作权法》

5.1.3《中华人民共和国计算机信息系统安全保护条例》

5.1.4《计算机软件保护条例》

5.1.5《互联网电子公告服务管理规定》

5.1.6《信息网络传播权保护条例》

5.1.7《中华人民共和国网络安全法》

5.1.8其他有关计算机及互联网规定的法律、法规。

### 用户在使用时，不得：

5.3.1制作、上传、复制、传送、传播包含任何反对宪法所确定的基本原则、危害国家安全、泄露国家秘密、颠覆国家政权、破坏国家统一、破坏民族团结、损害国家荣誉和利益、煽动民族仇恨、民族歧视、破坏民族团结、破坏国家宗教政策、宣扬邪教和封建迷信、淫秽、色情、赌博、暴力、凶杀、恐怖或者教唆犯罪、侮辱或者诽谤他人，侵害他人合法权益的等法律、行政法规禁止的内容或其他令人反感的包括但不限于资讯、资料、文字、软件、音乐、照片、图形、信息或其他资料；

5.3.2以任何方式危害未成年人；

5.3.3冒充任何人或机构，或以虚伪不实的方式谎称或使人误认为与任何人或任何机构有关；

5.3.4伪造标题或以其他方式操控识别资料，使人误认为该内容为HowToCook所传送；

5.3.5将无权传送的内容（例如内部资料、机密资料）进行上载、张贴、发送电子邮件或以其他方式传送；

5.3.6将侵犯任何人的专利、商标、著作权、商业秘密等知识产权的内容加以上载、张贴、发送电子邮件或以其他方式传送；

5.3.7将广告函件、促销资料、"垃圾邮件"等，加以上载、张贴、发送电子邮件或以其他方式传送；

5.3.8跟踪或以其他方式骚扰他人；

5.3.9将有关干扰、破坏或限制任何计算机软件、硬件或通讯设备功能的软件病毒或其他计算机代码、档案和程序之资料，加以上载、张贴、发送电子邮件或以其他方式传送；

5.3.10干扰或破坏GitHub服务或与GitHub服务相连的服务器和网络，或不遵守本协议之规定；

5.3.11故意或非故意违反任何相关的中国法律、法规、规章、条例等其他具有法律效力的规范；

5.3.12用户需保证其上传节目不得违反广电总局的相关规定，包括但不限于《互联网视听节目服务管理规定》（广电总局56号令）等，其上传节目应当符合法律、行政法规、部门规章的规定，上传节目不得含有以下内容：

5.3.12.1 反对宪法确定的基本原则的；

5.3.12.2 危害国家统一、主权和领土完整的；

5.3.12.3 泄露国家秘密、危害国家安全或者损害国家荣誉和利益的；

5.3.12.4 煽动民族仇恨、民族歧视，破坏民族团结，或者侵害民族风俗、习惯的；

5.3.12.5 扰乱社会秩序，破坏社会稳定的；

5.3.12.6 诱导未成年人违法犯罪和渲染暴力、色情、赌博、恐怖活动的；

5.3.12.7 侮辱或者诽谤他人，侵害公民个人隐私等他人合法权益的；

5.3.12.8 危害社会公德，损害民族优秀文化传统的；

5.3.12.9 有关法律、行政法规和国家规定禁止的其他内容。

5.3.13如用户提供的上传节目含有以上违反政策法律法规的信息或者内容的，由用户直接承担以上导致的一切不利后果，如因此造成不利后果的，用户应负责消除影响，并且赔偿因此导致的一切损失。

**Comment by wdzeng** [comment]
![https://www.youtube.com/watch?v=Yr7f7yMf0Tw](https://i.ytimg.com/vi/Yr7f7yMf0Tw/maxresdefault.jpg)

**Comment by fan87** [comment]
is saying "google it" against the rule

**Reply by wdzeng** [reply]
> is saying "google it" against the rule

Yes, because using Google is prohibited in the great country. It violates the rules 5.1.1, 5.1.2, 5.1.3, 5.1.4, 5.1.5, 5.1.6, 5.1.7, 5.1.8, 5.3.11, 5.3.12, 5.3.12.1, 5.3.12.2, 5.3.12.3, 5.3.12.4,  5.3.12.5, 5.3.12.8, and 5.3.12.9.

**Reply by TecReaGroup** [reply]
Hhha, don't mind,  just a disclaimer because he live in CN

**Comment by oiOxOio** [comment]
![B OKCK(OY9JL76JS99AXTU6](https://user-images.githubusercontent.com/76860802/180898829-53fb125c-1dfe-4fa1-9a7b-a827209af6a7.jpg)

**Comment by jacksmith1698** [comment]
好好好

**Comment by s9f3** [comment]
人才啊坐着
